components/TextFields/tests/snapshot/MDCBaseTextFieldTestsSnapshotTestHelpers.m

```diff
@@ -27,6 +32,43 @@
  */
 @implementation MDCBaseTextFieldTestsSnapshotTestHelpers
 
++ (MDCBaseTextField *)createBaseTextField {
+  CGRect frame = [MDCBaseTextFieldTestsSnapshotTestHelpers defaultTextFieldFrame];
+  MDCBaseTextField *textField = [[MDCBaseTextField alloc] initWithFrame:frame];
+  textField.borderStyle = UITextBorderStyleRoundedRect;
+  [MDCBaseTextFieldTestsSnapshotTestHelpers prepareTextFieldForSnapshotTesting:textField];
+  return textField;
+}
+
++ (MDCFilledTextField *)createFilledTextField {
+  CGRect frame = [MDCBaseTextFieldTestsSnapshotTestHelpers defaultTextFieldFrame];
+  MDCFilledTextField *textField = [[MDCFilledTextField alloc] initWithFrame:frame];
+  [MDCBaseTextFieldTestsSnapshotTestHelpers prepareTextFieldForSnapshotTesting:textField];
+  return textField;
+}
+
++ (MDCOutlinedTextField *)createOutlinedTextField {
+  CGRect frame = [MDCBaseTextFieldTestsSnapshotTestHelpers defaultTextFieldFrame];
+  MDCOutlinedTextField *textField = [[MDCOutlinedTextField alloc] initWithFrame:frame];
+  [MDCBaseTextFieldTestsSnapshotTestHelpers prepareTextFieldForSnapshotTesting:textField];
+  return textField;
+}
+
++ (CGRect)defaultTextFieldFrame {
+  return CGRectMake(0, 0, 200, 60);
```

Comment:
would this be able to capture multi-lines? should this use sizeToFit?

Reply:
`-sizeToFit` is called on the textfield before validation. This is just an initial frame.